pkgs/development/libraries/libcutl/default.nix

```
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
     license = licenses.mit;
   };
 
-  majmin = builtins.head ( builtins.match "([[:digit:]]\.[[:digit:]]+)\.*" "${version}" );
+  majmin = builtins.head ( builtins.match "([[:digit:]]\\.[[:digit:]]+)\\.*" "${version}" );
```

```suggestion
  majmin = builtins.head ( builtins.match "([[:digit:]]\\.[[:digit:]]+).*" "${version}" );
```

That should fix the evaluation. It does go to show that these constructs are dodgy. "\\.*" would be too strict to match the rest of the sequence, in this instance ".0"